### ISSUE TITLE
Mutation design discussion/planning

### ISSUE THREAD

**Issue by I-am-Erk** [CleverRaven/Cataclysm-DDA#71048]
### Is your feature request related to a problem? Please describe.

In many ways this is a followup to #28277, which was partially implemented but not entirely, now going over the existing issues with the revamped mutation system.

After reviewing several PRs, issues, and even posts in communities I try to avoid, I can see a number of issues with mutations. I am not going to exhaustively list them all but a few, in no specific order, are:
1. Terminology and nomenclature is extremely confusing and the whole system is hard to understand.
2. Negative mutations still don't fill any logical play role. They either end your game and you savescum, or they can be ignored.
3. It's still not entirely clear from a design standpoint what mutations are supposed to be, and when we want people to get them.
4. Related to 3, it's quite hard to get started mutating until you're already into the end game; contrast to CBMs, which are fairly easy to get early on, and are supposed to move more in that direction over time.

**Please note** this is a WIP issue. I will continue to edit and polish it as I go. If you find something confusing or think it will cause unexpected problems, let me know and make your case; I've already changed this quite a lot based on feedback in Discord. Also note that just because you make your case doesn't mean I'll *definitely* change this, but I do promise to listen. In return, I'd appreciate if you make sure you've read the whole thing before listing your criticisms. Thanks!

### Solution you would like.

There are several phases of solution here. I'll start with things that can be done pretty quickly to help resolve issues with the game as it presently is, then move on into stuff we can do in the near-or-mid future, and general design discussion that should get translated into the design doc once I'm happy with it.

I will disclaim here that I have played the mutation system as it stands, but I haven't done what I'd call a long term 'mutant playthrough' much in this system. I am very open to feedback on alternative ideas. These recommendations are a summary of a lot of conversations with others though, not just my own.

### Early stage stuff - basic summary
We're going to work on confusion and terminology by making sure the words we use mean clear things. There are two ways you can get mutagen:
- `one-stage mutagen` (1SM) is the basic mutagen, the stuff you drink and then you mutate. In the pre-revamp system this would have been called "mutagen" and two-stage mutagen has evolved from what we'd have called "serum" in those days. On the player side, if you drink 1SM you get a couple mutations, it's that simple. You can Do Science To It to refine it down to the components of 2SM.
- `two-stage mutagen` (2SM) comes in two items, neither of which is called "mutagen" on their own, so that it's easier to keep these separate. The two stages of 2SM are `primer` and `catalyst`, like they were before, but the way these work is going to fit the descriptions better.
- `anti-mutagen`, previously known as "chelator", can be used to stop the mutation process. It should be very easy to obtain and to craft, largely for play balance reasons.
- There should be more ways to obtain mutation products in limited amounts early in the game, to allow players a chance to get a taste for mutations like we do with CBMs.

Now without further ado, let's get verbose.

#### Part one: Catalyst and antimutagens

There was a lot of waffling on what to do with catalyst but after a lot of thought, I think the solution here is actually just to make catalyst into what we think of as a catalyst in chemistry terms. In other words:
- when you take a dose of catalyst, it drops off very slowly on its own as your body digests it, it might take weeks to get rid of it
- Mutating does not reduce the level of catalyst, so if you take a dose of catalyst, you could keep mutating for weeks.
- the more catalyst you have, the faster you mutate; you can add catalyst to one-stage mutagen, if you want to mutate faster. 
- This probably also means you'll get more mutations because your mut vitamin level won't have as much time to fall. This in turn means you won't need as many gallons of mutagen to mutate.

Special exception: After reaching a certain low end threshold level of catalyst it starts dropping off faster. This is necessary so that if you take one-stage mutagens, which add a bit of catalyst of their own, you don't just keep mutating. We'll get more into that in part 2.

Antimutagen (formerly chelator) should be easy to craft and labs should have tons of it since it was necessary for experimental controls. It reduces your catalyst level rapidly, though not instantly - if you took waaay too much catalyst it might not kick in in time to stop mutation.

Once you have "committed" to starting to mutate, you keep mutating even if your catalyst drops off; antimutagen just stops you from committing to any new mutations.

##### Perils of catalyst
In our early implementations of this system, there's not a lot of reason to not just leave catalyst on board for weeks and weeks. I don't think that's that big of a problem, but I do think we could add some minor penalties for having catalyst vitamin levels without mutating for too long. Joint cramps, headaches, etc. come to mind. It's also possible that having catalyst active without enough primer to trigger changes could cause you to develop random, mostly negative mutations over the very long term; I don't generally like negative mutations as a stick though, so I'd prefer we just have an annoying "catalyst syndrome" that kicks in after having moderate to high levels of catalyst but is temporarily removed by mutating.

##### Name confusion
With the changes in how it works, the name 'catalyst' should be less confusing and we can probably leave it here. The full name in lab notes and item descriptions could be longer, like "catalyst for two-stage mutation system". Digressing for a sec, I've seen a lot of weird takes on what is and is not a catalyst, but what I've described here fits the word just fine. If you are looking for a basic chemistry lesson hit me up on discord and I'll cover the details.

"chelator" is a great word but it was always a bad choice here, but in one of the many threads about this someone suggested 'anti-mutagen' and I agree that's probably much clearer. The only concern I can think of is that people will think this is going to work similarly to purifier; please comment on your thoughts.

##### A note on item categories
- The item category for these things should be changed to something like "Weird Science" or "Experimental Drugs", to reduce the habit of calling catalyst, primer, and unrefined mutagen all "mutagen". Catalyst should not be called mutagen, nor should primer. We *could* call the combination of them "two-stage mutagen", but only in documentation and descriptions, not on the item names.

#### Part two: "one-stage mutagen" (1SM)
The space currently occupied by various low grade mutagens could be simplified and broadened a little, into around 5-6 1SM types. These would be broad category mutagens, like "mammalian mutagen", "invertebrate mutagen", or "alien mutagen", each of which could access low end mutations from a couple different mutation trees. The exception would be purifier, aka human mutagen, which works like purifier. This isn't strictly necessary but I think there's not really a lot of need to have dozens of simple mutagens and also purifier and also untyped mutagen; we can ditch untyped mutagen and have all 1SMs have a broader type.

Like before, these 1SMs add several vitamins, both for the different mutation lines and for catalyst: by adding both, this means you can take just this stuff and you'll mutate, no extra ingredients, hence "one stage". (These vitamins are confusingly called mutagen in the code, I'm going to call them mut vitamins and cat vitamins). We need a little extra code here to set a cap for the amount of mut and cat vitamins you can get from consuming unrefined mutagen: after a few doses, there should be no more increase in vitamin levels no matter how much you drink, it just passes through you. 
- The cap level should be enough that you can mutate several times if you drink several doses.
- The cat vitamin should be set so that if it's at or below the cap level, it falls off quicker naturally. 
- The mut vitamin cap should be set too low to cross threshold.

The idea here is that 1SM can get you started on exploring some mutations, but is generally more random and limited than you'd like to try to get to threshold.

- note that these may be the only thing in game that should be called "mutagen" directly, outside of item descriptions. When someone says they're "taking mutagen" we're hoping to make it clear that they're drinking 1SM.

#### Part three: Primers

With the above changes I don't think we need to immediately change all that much about primer, although we might need to reduce the amount of mut vitamin it gives you since you'll no longer be limited by how much catalyst you take. I'm not sure of that though, it might be okay for a single dose of primer to be enough to send you wildly mutating like crazy until you take anti-mutagen.
- I do want to change the passive buffs given by primer. These should now only be something you can get if you take primer for a mutation category you have already threshed in, not available to pre-thresh mutants. The role of "weird drugs I take to get buffs" should be filled by a different thing (super solder serums), not mushed in with mutation. However those serums probably shouldn't work on post-thresh mutants (medical mutants could be a special exception), so primers can be a bit of a replacement.
- Post-threshold mutants should passively produce a small amount of mut vitamin in their primer type, so if they chug unrelated 1SM mutagen or even if they just take catalyst on its own, they have some chance to mutate within their own category - but this should be a low background amount. You should need a lot of catalyst to get any spontaneous mutations.

In general we should review the crafting and obtaining of primers and catalysts to make sure that mutation is accessible and fun without being too easy. Personally, I'd like the crafting to be hard mid-game stuff, but we could really use with some ways to get ahold of small amounts of ingredients prior to looting labs. This falls more into the lore category, eg. we could have some mission chains that help you get access to smuggled mutagens that made it out of the labs, enough to try them out but not to fuel your whole mutation chain. Or the Hell's Raiders could have a black market that sells small amounts of mutagen.

#### Medium to long term changes: General design thoughts.
1. The role of transhumanism, especially mutation, hasn't been well marked out for a while (except for their roleplay/lore value) and it is part of why these are getting tangled up. I think we need to be clear on this. Specifically, I think it's important to emphasize that the design goal for mutation trees is *not* to turn you into a posthuman powerhouse. That's also not the design goal for CBMs. In general, a mutant or cyborg is going to find the late game easier than a normie, but the difference shouldn't be all that stark: players shouldn't feel like they *have* to choose a transhuman line to progress, although they should feel tempted.
    - Looking at mutations vs CBMs in specific, in general CBMs are likely to feel more powerful, especially since you can choose your CBMs. Again, this shouldn't be night-and-day, but it is OK if cyborgs are a bit stronger than mutants in the long run. This is reasonable in the lore since there's a faction trying to get you hooked on cybernetics for their own benefit.
    - Looking at mutations as a whole I think the play element that we need to focus hardest on is that they should, more than CBMs and much more than equipment, *fundamentally change the way the game is played*. There are a few exceptions to this, like the humanoid mutation trees, but generally you should decide to cross threshold because you *plan* to uproot some of the more fundamental gameplay aspects.

When we look with these points in mind I think it gets easier to decide how we should develop mutations in the future. The 'fundamental play shift' part means we should look into ways to make threshold-crossing something the player can do earlier, even if we need to do it through meta means (like a metaprogression that unlocks the ability to start as a threshold mutant in a new playthrough) although I think that's the least satisfying solution. It also means that it is quite acceptable for mutation trees to limit access to basic aspects of the game, like driving vehicles or wearing armour, but *the mutation tree should offer alternatives*. They don't have to be *equal* alternatives! If growing horns keeps you from wearing helmets we don't need to also offer equal head armour, night vision, or helmet utility, but we should be sure to keep an eye on how much is being lost and try to give some interesting counterpoints that can't be obtained outside the mutation tree that gives you horns. Chances are, most of the time they're not going to be as versatile and therefore powerful as wearing gear would be, but by making them unique then there's an interesting play choice and variety to be had on future plays.

This leads me nicely to:

2. Random "negative" mutations are a terrible play balance element. They are easily circumvented in boring ways by save scumming, and getting them rarely makes the game more fun. Horns in the above example are a generally negative thing because they block helmets. In the current system if you want to wear helmets, you can just cheat away until you lose your horns one way or another. As we see above, this can actively harm our ability to balance the design of mutation trees, because we basically need to assume players aren't getting negative mutations.
 - I think the solution to this has to be linking positive and negative mutations together, especially the more powerful positive mutations. Horns might come along with tough lizard skin automatically, you can't have one without the other. Disintegration shouldn't be a thing at all, but a dramatically nerfed version of it might be a consequence to the highest levels of alpha strength. This ties to tradeoffs as listed above... rather than a straight positive change, you have to weigh the positive with the expected negatives. It also reduces the randomness. When you mutate you should get something that is *interesting, always*. It might not be *powerful*, and it might not be what you're going for, but in general we'd like people to rub their chins and consider how this is going to change how they interact with the game.

#### Long term thoughts
This is where I go into the "mutations as blackjack, not roulette" stuff again, but I have to do real life work so I'll write it up and post it in shortly. In very brief, this is the idea I had way back in the old revamp, where instead of a bunch of individual random mutations we link mutations together into longer trees, and once you've committed to a mutation we start mutating you up that tree, not adding new random mutations, until such time as you take antimutagen and halt the process. The higher you go along a tree the stronger the positives and negatives both become, allowing you to make the decision whether to "hit" or "stay" by taking, or not taking, antimutagen.

### Describe alternatives you have considered.

So many. So, so many. I'm tired of thinking of alternatives.

I'll write some up or link discussions here in a bit.

### Additional context

I know there are people who feel unheard throughout this process, but I really want to emphasize that this represents many days of carefully reading and considering all the conversations I and others could find about this. Remember that just because someone doesn't come to the same conclusion as you, doesn't mean they're not listening to you or even that you aren't changing their mind.

**Comment by Venera3** [comment]
I don't think the free-floating negative mutations are a net loss for the current system, and doing hard-linked pairs kind of defeats the point of having two traits - just slap the drawback on the main one if it fits. Having the option of growing something unexpected is the main flavor of mutations and it sets them apart from the über-determinism of bionics.
 I could see each bigger trait roll from something like three-five mutually exclusive drawback traitlets as an alternative, though.

I am on board with the perma-catalyst, though it's not a million miles off of what the current system achieves - the problem is of presentation and the "type mutagens" being a trap setup. Catalyst decay mostly stems from the failed mutation roll and lives in the changing EoC, for easier tweaking.

**Comment by fairyarmadillo** [comment]
I'm gonna be pointing two two reddit posts here. One is by contributor @anoobindisguise . It's slightly outdated, but still illustrative: https://www.reddit.com/r/cataclysmdda/comments/118on78/

The above guide allowed anoobindisguise to get dozens of beneficial or neutral mutations with almost no downsides. I consider that a problem, but even worse, it's _boring._ The current system's most pressing issue is that it allows you to get very deep into the mutation system without ever once getting a red mutation, and the way that you do this is by standing in a field with some water and pemmican and pushing wait. Because of this, many players feel that they can't even begin mutating until they have access to Alpha (again, see the guide for why), and the resulting mutation process involves simply waiting out the instability timer and taking very specific doses.

For a lot of players, because this totally safe and deterministic way to get maximum upsides and minimum downsides exists, no other method is acceptable, even if this one is so obnoxiously time consuming that they won't actually do it. The result is either: People follow the guide and have a very boring time, or they don't and never engage with the system.

Most of the complaining about negative mutations comes from people who admit to totally avoiding the system, and I'm not sure how much we should worry about that. I watched a someone warn new players not to engage with the mutation system because paws have a -10 hand encumbrance and heat-dependent is apparently a "run ender" because it slightly penalizes your speed sometimes - maybe we should just accept that that person doesn't want or need to play a mutant if that's a dealbreaker for them.

Anyone who's played a mutant can tell you that even extreme stuff like gastropod foot, disintegration, and ectothermic are not going to stop a creative player from being successful at the game. You certainly have to play around extreme penalties, but that's where most of the flavor of being a mutant comes from, and personally that's why I think they're more interesting (and certainly a deeper system) than bionics.

The other post is by me: https://www.reddit.com/r/cataclysmdda/comments/197eklf/why_dont_you_play_a_mutant/ I asked why people don't play mutants and got over 200 replies. Mostly, it seems like since the wiki isn't updated anymore and the process is a lot more confusing than it used to be, people don't want to touch the black box for fear it'll be an immediate end to their character.

TLDR: I agree with Venera above. Standalone red mutations are perfectly fine and even good, and the only problem I see with them right now is that it's possible to completely avoid them while still getting lots of green mutations. We should change instability to instead ensure a proper ratio of good to bad muts. Also we could maybe encourage people to work on the wiki.

**Comment by I-am-Erk** [comment]
> I don't think the free-floating negative mutations are a net loss for the current system, and doing hard-linked pairs kind of defeats the point of having two traits - just slap the drawback on the main one if it fits. Having the option of growing something unexpected is the main flavor of mutations and it sets them apart from the über-determinism of bionics. I could see each bigger trait roll from something like three-five mutually exclusive drawback traitlets as an alternative, though.

It's unclear in how I wrote it up but I am not really picturing the up-and-downsides being in linked pairs, they'd probably generally be part of the same mutation yeah. The only reason to have them function in pairs is if we want to have a mutation line where there are several different possible negatives that can be mixed and matched, which might be interesting but also might lead us back to 'savescum until I get the least bad one' so I wouldn't go there without good cause.

> I am on board with the perma-catalyst, though it's not a million miles off of what the current system achieves - the problem is of presentation and the "type mutagens" being a trap setup. Catalyst decay mostly stems from the failed mutation roll and lives in the changing EoC, for easier tweaking.

A lot of what I've written isn't meant to be a huge uprooting of the system, I don't think the current system is terrible... just obtuse and challenging.

**Comment by carlarctg** [comment]
I want to be able to begin the mutation process at the midgame. I've got makeshift armor, a working truck, supplies to last me for a month or two, and am actively doing missions for a faction. By now I should be able to start mutating in a rather reckless and careless way, somehow, so that by the time I become a posthuman my mutation process FINISHES, rather than starting. I want mutations to be have several gates throughout progression, not just 'the moment you find mutation supplies you're done, and also you are done with the game and there's no challenges left anyways so why bother'

I want there to be a phase where my character can start wildly mutating, flesh pulsing and oozing, one inbetween absolute nothingness and the careful tedium of science and fine distillation and concentration.

Midgame - Begin mutating (if interested)
Lategame - Do some quest or whatever, put some elbow grease in, get post-threshold. Thrive.

Our survivor is apparently blobbed up and has a mild amount of feral psychosis. Lean into that. Tap into it. Have a base of nutjobs that through a mixture of completely fake hedge magic and actual stimulus that blob reacts to, will help you interact with your inner mutation-capable blob, without demanding you venture into the endgame laboratory dungeons.

It would also be nice if there were scenarios/jobs/professions that could speed this up for those that want to directly play a 'mutation run', without the cheapskate experiment mutation-traits that don't even help you get any future mutations. I have vague plans in mind for a trait that makes you slowly mutate in accordance to your environment, diet, and other stimulus, something like that.

The old, old system was better for this because you could find some mutagen, take it, get cool mutations. Now by the time you do have some mutagen you'll likely have a full laboratory setup for everything you could ever want as well. (I may be speaking out of my ass here)

**Comment by I-am-Erk** [comment]
Please read the first post carlarc, I mentioned those specific concerns in a couple different spots.

**Comment by carlarctg** [comment]
I _have_ read it, read both the posts even, but I don't see how it invalidates my post's points? You've spoken a lot about how the system should change, but little on when in the player's playthrough should they begin experiencing said system.

**Comment by fairyarmadillo** [comment]
> I'm really not sure if this would solve anything, or if that has been suggested before (it probably was), but...

Practically speaking, that would just lead people to view that item as a mandatory condition/tax for mutating at all.

I don't think there should be a way to avoid the downsides. The downsides are how we sell the dramatic impact of the mutations and force players to approach the game differently.

**Comment by fairyarmadillo** [comment]
I don't want to spam the thread, but if you reread my comment above you can see why I think that's a bad idea.

**Comment by I-am-Erk** [comment]
> The 'fundamental play shift' part means we should look into ways to make threshold-crossing something the player can do earlier, even if we need to do it through meta means (like a metaprogression that unlocks the ability to start as a threshold mutant in a new playthrough) although I think that's the least satisfying solution.

EG:
"There should be more ways to obtain mutation products in limited amounts early in the game, to allow players a chance to get a taste for mutations like we do with CBMs."

"The 'fundamental play shift' part means we should look into ways to make threshold-crossing something the player can do earlier, even if we need to do it through meta means (like a metaprogression that unlocks the ability to start as a threshold mutant in a new playthrough) although I think that's the least satisfying solution."

I'm not invalidating what you're saying, I'm saying this is already something I've said we'd like. Feel free to suggest how you'd like to achieve it.

**Comment by I-am-Erk** [comment]
>So I'm guessing then we'd need to have every non-trivial red mutation also do something "cool", right?

It's not so much that, though that's close You're basically touching on why I don't think "negative" mutations are all that great. Rather, every mutation should do something. They don't have to be balanced to "mostly good" or "mostly cool" but the system works better in general if either:
- negative mutations have a chance to progress into something better, like the current change between "freakishly huge" --> "huge", or
- some mutations have drawbacks that probably outweigh the benefits but they're still focused more on the good things that they do.

So for example, using the bovine track, the "deformed" mutation might still exist but it would be step one on a mutation line that progresses further towards "bovine snout", but now bovine snout has some positives: you're getting a full bovine digestive tract, so the benefits of 'ruminant' have been merged in as well (makes sense, you need cow teeth to eat grass safely as well). In this example, yeah deformed is still straight bad; snout is now balancing a bad trait with a new positive, and the next step in the mutation would further your facial changes but also improve your gi changes towards being a full grazer.

**Comment by DragonWizard23** [comment]
> negative mutations have a chance to progress into something better
> So for example, using the bovine thresh, the "deformed" mutation might still exist but it would be step one on a mutation line that progresses further towards "bovine snout", but now bovine snout has some positives

I like this idea because it leaves room for negative mutations without punishing the player. Instead, it encourages the player to continue mutating towards more beneficial mutations. A similar case could be in the feline line, where "snout", which is both visible and ugly, could unlock "fangs", which gives the player some benefit and adds some visibility and ugliness too. The drawbacks remain, but the player is rewarded for mutating further. 

*P.S. I'm aware that fangs aren't currently locked behind snout, but I think the idea sounds cool. Besides, I don't think 2-inch-long fangs would fit into a human mouth. *

**Comment by anoobindisguise** [comment]
so a couple things I wanted to add:
- with regards to negative mutations, IMO anything that has even a tangential benefit should be neutral. I've PR'd mutations in that direction before. "Negative" mutations should be purely, 100%, negative with no upsides.
- Instability shouldn't be a decaying quality. Your instability is how many mutations you have, period, and can only be "reduced" by purifying. The more mutations you try to add on top of what you have already, the greater the odds that something goes wrong. This is similar to your "blackjack" proposal I think. I would exclude negative mutations from this calculation.
- One way to reduce the "sting" of negative mutations would be to bundle them with positive mutations. So say you're a Medical mutant and you're mutating while already heavily mutated, you get your Very Quick Pain Recovery like you wanted but now you're narcoleptic too or whatever. Since currently getting a negative mutation is a "double fuck you" since it hurts you and increases your instability on top of that. Again, like your blackjack proposal.
- Trying to mutate "out of category" (firing up a second tree after you've already gotten into a first) should cause that second tree to act as being "more unstable" the more mutations you have that don't belong to it. So say you have 10 mutations in Medical and you have a 15% chance of getting a negative mutation due to your instability (or whatever). You start taking Feline mutagen, but you have a 30% chance of getting a negative out of Feline. So a per-tree instability calculator that counts all your positive mutations, and counts doubly mutations that don't belong to that tree.
- For some mutation trees, their "negatives" are a fundamental part of the tree's identity. For example Trog's sunlight aversion. I'm not sure what the best way is to reconcile this with the "bad mutations come from you getting too reckless" idea. For Trog specifically we can make some of their post threshold super mutations (extremely powerful adaptions to cave life) dependent on their negatives, but that feels a bit lazy.
- Question, if we have "pseudo-typed" mutagens which are common but relatively untargeted and _also_ have traditionally typed mutagens, why would anyone take for example "mammal" mutagen if what they really want is "feline" mutagen? I'm hesitant to believe anyone would take an option that is worse but more readily available instead of just waiting for the better option because that's already how everyone behaves.
- Chelator should definitely be far more common.

**Comment by I-am-Erk** [comment]
too busy for a full reply right now but popping in just to note that I really like the idea that instability increases if you take out-of-tree mutations. I think in general it could increase based on number of trees you have mutations from, regardless of threshold, and then if you cross threshold, out-of-tree mutations could add a little bit more.

**Comment by fairyarmadillo** [comment]
> If we don't simply get rid of negative mutations, true negative mutations could actually _increase_ stability. 

This would be ideal IMO. You want the prize, you gotta pay the toll. If you are really clever, you might be able to find a way to end up with the least worst downsides, but you'll still have them, and we can avoid there being one sure path to a single ideal result.

As long as there's even a shred of risk, some percentage of people will savescum, but that's also true for people stepping on landmines or getting torn apart by hordes.

**Comment by IdleSol** [comment]
I apologize for the quality of the text. Deepl translate is used.

I'll start with a warning. I am one of those players who do not use mutations. I don't like randoms. And I apologize in advance if I suggest things implemented in the game.

Some ideas.

**1. Tiers for mutagens**

- Tier 1. Mutagen or 1-SM. Multiple random mutations.
- Tier 2. A mutagen of a specific line. Alpha, beast, etc. A human mutagen, aka purifier.
- Tier 3. A mutagen of a trait. Mutagen of night vision. Mutagen of branching horns, if anyone misses them. Why not? Long, expensive, and welcome to the labs.

**2. Revision of parameters subject to mutation**

Conventionally, a human can be described by a set of parameters. For example: fall damage modifier, melee attack rolls and others. We can assume that each of them is multiplied by a coefficient = 1, which corresponds to a human?

What happens if mutations change that coefficient? (Different for each parameter).

For example encumbrance. For a human, the penalties start at 10, but for a mutant of a certain line, will start at 20. Or not at all (slime).

**3. 2-SM, catalyst and anti-mutagen**

If we use a catalyst, then why not use an inhibitor?

And according to their names they should speed up and slow down the mutation process.
- Catalyst, accelerates, but causes many side effects: pain, depression, increased caloric expenditure. It is not excreted from the body.
- Inhibitor, on the contrary, slows down, but the mutation occurs more smoothly and without side effects. Expended along with the mutagen and/or excreted from the body. 
- Anti-mutagen - removes mutagen.
- Anti-catalyst - removes the catalyst.
- Anti-mutagen + Anti-catalyst = x10 side effects.

Given a tier, we get three types of one-stage mutagen:
- 1-SM-T1;
- 1-SM-T2-line_name;
- 1-SM-T3-trait_name.

It can be used as is. Or you can take it together with a catalyst or an inhibitor. And two-stage mutagen should be two different drugs, not one new drug. First of all, so there are fewer recipes, and secondly, so it is more realistic. In my opinion.

>the more catalyst you have, the faster you mutate; you can add catalyst to one-stage mutagen, if you want to mutate faster.

It is better to use the ratio of mutagen to catalyst. The closer it is to 1, the faster. Excess of catalyst over mutagen should not increase the speed.

>The only concern I can think of is that people will think this is going to work similarly to purifier; please comment on your thoughts.

The important thing is not the title. The important thing is the detailed description. If it says what it does, it doesn't matter what it's named.

**4. How is the player supposed to know how to properly use "human evolution preparations"?**

I don't really like this way, but you can write in the description of the mutagen.
Rabbit Mutagen. Method of consumption:
- 250 ml once a day. Course of reception 7 days. Side effects: pain, nausea and dizziness.
Method of reception together with catalyst (250 ml):
- 250 ml of the drug twice a day, course of 4 days. Side effects: severe pain, nausea, weakness and hallucinations.

And so on, as far as the imagination goes.

**Alternative 1**: Add a note from a scientist with a formula. The player has to substitute the values himself. And get how much and what he should consume. Something like:
```
1 dose = ceil(Weight/10)*50 ml     ; For 80 kg, this is 400 ml

Time to mutation = 12*(2 - min(1,catalyst/mutagen))   ; hour
catalyst = 250 ml
mutagen = 400 ml. 
Total: 12*(2 - 250/400) = 16.5 hours.
```
I hope I didn't get anything wrong.

**Alternative 2**: From recipe books.
As a basis we take a book of spells, where each spell is a description of the line of mutation, method of reception and dosage. And by description I mean either a direct listing of possible traits, or a more artistic version. In general, so that even a novice can understand what he will get and how to achieve it. Instead of wondering why he's now 3 meters tall and can't eat meat.

We will probably lose the opportunity to learn from this book, but there are enough books for applied science as it is.

Additionally, you can make different ways of taking and dosage for each mutation. And if you also make the dependence on a person's sex, height, weight and blood type... There are scales in the game for a reason.

**5. Portable blood analyzer**

Portable, not bionic. The item can be found in a hospital, laboratory. You can find a blueprint and create it (less restrictions for players).

In the basic version gives access to blood analysis and the amount of common vitamins.
You can find firmware (sd or usb) in the lab. The improved version should show the content of matagen, catalyst and everything else that can be seen through the debug menu. Preferably with display of dynamics, at least for a day.

**6. Mutation and bionics**

I have a biased opinion. I believe that mutations should lead to rejection and/or breakage of some bionics (not all). Or the inability to install. It all depends on the order. If the bionics are before the mutation, then it can be broken. If after, it can't be installed.

**7. A way to mutate at the beginning of the game.**

Yugg and his kind of creatures. This is another reason why I don't like mutations.

Find one, survive the attacks, and mutate. Or kill it, butcher it to pieces, and get the mutagen gland.

**9. New traits.**

Traits that will change the speed and amount of drug needed to mutate. As well as the chances of random mutation. Strong genome, weak genome, normal genome.

This would be insurance for those who don't like mutations, in case Yugg comes back.

You can replace the trait with an option in the game settings.

**10. New ways to turn into a posthuman.**

Off topic, but the game needs new ways. Not just mutations and bionics. And new challenges to overcome. Especially in the late stages of the game.

**Comment by IdleSol** [comment]
**11. Mutagen for pets**

New lines for horses, dogs, cats.
- A dog that can actually kill a mob of zombies, not freeze somewhere far away.
- A horse that moves at 60 km/h and carries 200 kg of weight. Loves meat, even if it's still walking.
- A cat is a small serial killer. After mutation will clear fields of aphids, birds and any other non-aggressive critter.
- A cow or bull is a living battering ram. Don't know how to break the Mi-Go wall? The answer is a bull.
- A hen. You shouldn't have given it the mutagen. Now something's after you. It's fast and hungry for fresh flesh. And there are more of them every day. Just for fun.

**Comment by Zireael07** [comment]
> There should be more ways to obtain mutation products in limited amounts early in the game, to allow players a chance to get a taste for mutations like we do with CBMs.

Yes PLEASE

**Comment by XygenSS** [comment]
Purifier should be called "human mutagen" explicitly or implicitly. The player can piece things together when they learn that creating mutagen requires a "sample" from dissecting the correct creature. Maybe also add, like, dessicated remains of a half-dissected corpse in one of the Lab operating tables.

**Comment by fairyarmadillo** [comment]
> Purifier should be called "human mutagen" either explicitly or implicitly. The player can piece things together when they learn that creating mutagen requires a "sample" from dissecting the correct creature. 

This is already how it works. We only call it purifier so that the player can realize on their own what it truly is.

**Comment by XygenSS** [comment]
> This is already how it works. We only call it purifier so that the player can realize on their own what it truly is.

Would it be too on the nose if it were a bit more obvious?

**Comment by fairyarmadillo** [comment]
Yes

**Comment by IdleSol** [comment]
>Yes. The way you find out is that you have to butcher human

What, now it's people that need to be butchered? Last time I tried it, it was feral. There was nothing to be taken from humans. It was a waste of time.

How does the feral genome match the human genome? I thought they were already blob-influenced. It was just a brainwashing.

**Comment by IdleSol** [comment]
I mean, the source of human samples needs to be reconsidered. Only humans. If it hasn't already been done.

The feral can provide their own sample, but not human. For example, for a medical mutation.

**Comment by IdleSol** [comment]
New sources of mutation
- organ transplantation;
- symbiotic organisms;
- parasite infestations.

I don't know what the plans are for the limb system. And whether individual internal organs will be injected. But it's already possible to start without arms and legs.

Does it fit into the game, the ability to transplant another person's limb, including mutants. Perhaps a zombie limb? But certainly not in the early stages of the game.

Intentional slime infestation? Or a new creature that needs to be captured and something done to get a symbiote.
And there are two ways to go about it:
- The symbiote serves as a source of mutation
- Symbiote is a suit (Marvel). Instead of mutating the character, there is costume evolution. Restriction on bionics, mutations (just blocks it). Restriction on protective gear.

To give you an example. 1d1000 to make the slime friendly. It likes you, i.e., you are very tasty. And he wants to take the place of your clothes. Stages of evolution - layers of clothing and % coverage.

Parasites, there is a chance to catch a parasite when butchering mutated creatures. You can get rid of it, or you can wait.

Translated with DeepL.com (free version)

**Comment by TheSaddestGoomba** [comment]
I have plans to add a set of worm-infested, mutant frog monsters based on the IRL frog-mutating flatworms. One of my notes is that these should be a nightmare for batrachian mutants, potentially infecting them with the worms. I had originally thought of it as just a potential bad end if the player doesn't cure it. I hadn't considered that anyone might want to keep them or form some kind of symbiosis with them once infected. I'm not sure when I'll get to it, but I like the idea of picking up a gross worm-buddy to accompany you in the apocalypse.

**Comment by fairyarmadillo** [comment]
> I mean, the source of human samples needs to be reconsidered. Only humans. If it hasn't already been done.
> 
> The feral can provide their own sample, but not human. For example, for a medical mutation.

I don't really understand what you mean. Ferals are human beings, no different from the player except in their mental state. The only reason you weren't getting samples from NPCs is because nobody's coded that. They should definitely be giving them.

Medical samples are artificially manufactured and cannot be harvested from anything except medical mutants. We don't have any of those in the game yet, but I'm adding some with #70771

**Comment by IdleSol** [comment]
>I don't really understand what you mean

Maybe it's a headcanon. 

The Blob infected all humans. As a result, most died, a small fraction survived. The feral ones are somewhere in between.

The Blob is the source of mutations, according to lore (or an older version of it). It changes the hosts. With the dead everything is clear, reanimation and gradual adaptation to new conditions.

The survivors were lucky. Although we can add a detail: the ability to learn and assimilate information has increased. That's the rationale behind the speed of learning from novice (0) to master of their craft (10), in a couple months.

I believed that the feral ones also had changes (or mutations, if you want), only they were not to the body but to the brain. Increased aggression, lack of critical thinking and so on.

Accordingly, their samples may be close to human, but they are not 100%. And therefore should not be used to make a purifier. But they can be used to make a medical mutagen. No pain. Head problems. Sounds exactly the same to me.

P.S. If you're referring to the quality of the text, I apologize.

**Comment by IdleSol** [comment]
>I have plans to add a set of worm-infested, mutant frog monsters based on the IRL frog-mutating flatworms.

1. When butchering spiders (if I'm not mistaken), direct text says something about something running away from you. Here's another source for parasites.

2. You don't have to make the parasite safe. Yes, it gives mutation, but it also kills. Sometimes suddenly. How far will you go in the pursuit of power? Can you stop in time?

3. Mind control. Take control of your character. Make him eat the supplies. Anything in the surrounding area. Regardless of depravity. Traits are ignored. Stomach overflowing. Pain. Nothing matters, just hunger and 5 minutes to satisfy it. Corpses are good too.

**Comment by fairyarmadillo** [comment]
> > I don't really understand what you mean
> 
> Maybe it's a headcanon.

That is almost all headcanon and not relevant to our plot. The blob "superpowers" are a shoddy handwave to explain code limitations which are in the process of being removed (see: proficiencies, the wound system), and there is no difference between the player and a feral, biologically speaking.

**Comment by IdleSol** [comment]
I focused on https://cddawiki.chezzo.com/cdda_wiki/index.php/Timeline 

>January 2025: The blob reaches satisfactory global-level groundwater saturation and makes the decision to commence the concentration and replication phase. For the next few weeks there is a gradual rise in violent crime, and occasional reports of strange mutations and weird animal sightings in the wild, but little attention is paid.

Not canon, so not canon.

I can't talk to them, interact with them. To me, they're no different than zombies.

NPS, I can talk to and interact with them. And I never attack first. Except for bandits. Personally, for me, it would be a problem to collect samples from NPS. Again, forgetting the bandits.

**Comment by I-am-Erk** [comment]
We're quite far I think from mutation design discussion here. I'd like to keep it on topic please, I've hidden a few posts that were cluttering things up. Note that this isn't anything against the posters or their ideas, it's just that this is already a long topic and will likely be longer before the day is done.
